### ISSUE TITLE
Register button style fix

### ISSUE THREAD

**Issue by Kaltsoon** [hh-ohjelmistoprojekti-2/spring-boot-vite-example#18]
The current register button styling violates the style guide and it has a poor contrast making it bad for accessibility. The button should have a white outline and invisible background.
